Doc/library/string.rst

```diff
@@ -574,11 +574,13 @@ The available presentation types for :class:`float` and
    | ``'%'`` | Percentage. Multiplies the number by 100 and displays    |
    |         | in fixed (``'f'``) format, followed by a percent sign.   |
    +---------+----------------------------------------------------------+
-   | None    | For :class:`float` this is the same as ``'g'``, except   |
+   | None    | For :class:`float` this is like the ``'g'`` type, except |
    |         | that when fixed-point notation is used to format the     |
    |         | result, it always includes at least one digit past the   |
-   |         | decimal point. The precision used is as large as needed  |
-   |         | to represent the given value faithfully.                 |
+   |         | decimal point, thus switch to the scientific notation    |
+   |         | happens when ``exp >= p - 1``.  If precision is not      |
+   |         | specified, the precision used is as large as needed to   |
+   |         | represent the given value faithfully.                    |
```

Comment:
It's a hassle to make suggestions in RST tables so I'll write the text here:

"past the decimal point, and switches to the scientific notation when ``exp >= p - 1``. When the precision is not specified, the latter will be as large as needed to represent the given value faithfully."

Reply:
(Sorry for reviewing it soooo late).

Reply:
Not a problem.

My major task (to figure out how it really works) was accomplished; the mpmath code for new-style formatting seems to be compatible with CPython.